### DISCUSSION TITLE
My user name end with a dash

### DISCUSSION THREAD

**Discussion by j-catania** [community/community#27186] [Actions]
<p>Hi Everyone,</p>
<p>I have an issue when trying to push an image to <em><a href="http://ghrc.io" rel="noopener nofollow ugc">ghrc.io</a></em> (this will be the same for all other registry).</p>
<p>Here is the issue:</p>
<pre><code class="lang-auto">invalid tag "ghcr.io/juu-/vivaio_backend:latest": invalid reference format
</code></pre>
<p>I looked in an issue in the <em>docker/build-push-action</em> project: docker/build-push-action/issues/237<br>
Like you can read in the feed, there is maybe a problem with my username because it end with a dash ‘<a href="https://docs.docker.com/engine/reference/commandline/tag/#extended-description" rel="noopener nofollow ugc">A name component may not start or end with a separator</a>’</p>
<p>Do you have some idea to fix this issue ?</p>
<p>Best regards,<br>
Juu’</p>

**Comment by whitneyimura** [comment, marked as answer]
<p>Hi  <a class="mention" href="/juu">@Juu</a> - The spec is correct and we don’t support the underscore in order to stay compliant with the spec.  My guidance to you, especially since you don’t have any containers published yet, is to rename your handle (if that’s an option).</p>

**Comment by j-catania** [comment]
<p>Hi <a class="mention" href="/whitneyimura">@whitneyimura</a>,</p>
<p>Thanks for your reply !</p>
<p>Changed my username yesterday. A little bit hard to refractor all where that was mentioned but it’s done now.</p>
<p>Thank you,<br>
Julien</p>
